frontend/public/images/dashboards/icon_commodities.svg

```diff
@@ -1,23 +1 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We can actually use https://github.com/tcoopman/image-webpack-loader to optimize SVGs with SVGO

Reply:
Nice! Though we'd need to test it works without importing the images like
```js
import myImage from '../images'
```

Reply:
We already have it on prod, but not sure it's working and it's probably due to how we load the svgs. Let's review that in the future.